src/Databases/DataLake/DatabaseDataLakeSettings.cpp

```diff
@@ -17,7 +17,7 @@ namespace ErrorCodes
 }
 
 #define DATABASE_ICEBERG_RELATED_SETTINGS(DECLARE, ALIAS) \
-    DECLARE(DatabaseDataLakeCatalogType, catalog_type, DatabaseDataLakeCatalogType::ICEBERG_REST, "Catalog type", 0) \
+    DECLARE(DatabaseDataLakeCatalogType, catalog_type, DatabaseDataLakeCatalogType::NONE, "Catalog type", 0) \
```

Comment:
Why do we need none catalog? At first I thought that this is in order to force this setting to be always specified by user explicitly, e.g. to not have a default value, but it does not seem to work this way. Moreover, this must be backward incompatible.